pkg/hyper/hyper.go

```diff
@@ -53,12 +52,18 @@ func NewHyperRuntime(hyperEndpoint string) (*Runtime, error) {
 		return nil, err
 	}
 
-	return &Runtime{client: hyperClient}, nil
+	return &Runtime{Client: hyperClient}, nil
 }
 
 // Version returns the runtime name, runtime version and runtime API version
 func (h *Runtime) Version() (string, string, string, error) {
-	return hyperRuntimeName, "", "", fmt.Errorf("Not implemented")
+	version, apiVersion, err := h.Client.Version()
```

Comment:
h.Version() will call Runtime.Version, so I rename Version to GetVersion in client.go